graylog-configurer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine
+
+COPY run.sh /
+
+RUN apk add --no-cache curl jq coreutils
+
+CMD sh /run.sh
```

Comment:
If you make `run.sh` executable, then this can be invoked `CMD ["/run.sh"] thus not spawning a new shell.

https://docs.docker.com/engine/reference/builder/#cmd